charts/credentialfeeder/Chart.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: v2
+name: credentialfeeder
+description: A Helm chart to generate keys
+type: application
+version: 1.0.0
```

Comment:
please get this version confirmed by @ckm007

Reply:
Confiemed the Chart vesion with @LoganathanSekar7627.